lib/internal/Magento/Framework/Mail/Address.php

```diff
@@ -29,7 +29,7 @@ class Address
      * @param string|null $name
      */
     public function __construct(
-        ?string $email,
+        string $email,
```

Comment:
I would suggest changing the `getEmail()` method instead of changing the constructor parameters. So we have the constructor untouched but

```
public function getEmail(): ?string
```

This solution will be more consistent considering the same approach in `getName()` method.

Thank you